Libs/MRML/DisplayableManager/vtkMRMLModelDisplayableManager.h

```diff
@@ -217,9 +216,6 @@ class VTK_MRML_DISPLAYABLEMANAGER_EXPORT vtkMRMLModelDisplayableManager
 
   vtkMRMLSelectionNode* GetSelectionNode();
 
-  /// Find picked node and cell from mesh and set PickedNodeID and PickedPointID in Internal
-  void FindPickedDisplayNodeFromMesh(vtkPointSet* mesh, double pickedPoint[3]);
```

Comment:
What do you think of adding an entry to https://www.slicer.org/wiki/Documentation/Nightly/Developers/Tutorials/MigrationGuide ?

Reply:
Should I rather add it back?

Reply:
Since we are moving Slicer 5.0 .. breaking things is fine. I suggest we keep a record.

Reply:
I added a new section for 5.0 changes and added this as the only item so far